### ISSUE TITLE
LVM physical disk not updated (pvresize)

### ISSUE THREAD

**Issue by Roxyrob** [mrlesmithjr/ansible-manage-lvm#58]
This role seems not to automatically extend pv after increasing disk size.

Reproduce steps:

Run a playbook using this role with this initial configuration (/dev/sdb = size 5GB)

```
manage_lvm: true
lvm_groups:
  - vgname: 'mysqlvg'
    disks:
      - /dev/sdb
    create: true
    lvnames:
      - lvname: 'mysqllv'
        size: '+100%FREE'
        create: true
        filesystem: xfs
        mount: true
        mntp: '/mysql'
```

Increasing /dev/sdb disk size (e.g. in aws ebs volume or in vsphere VM virtual disk) and running again, does not extend pv on the new disk size, subsequently leaving vg, lv and filesystem not espanded.

**Comment by Roxyrob** [comment]
lvg module in Ansible >= 2.10 using collection add support for pvresize (pvresize = If yes, resize the physical volume to the maximum available size.)

[community.general.lvg](https://docs.ansible.com/ansible/3/collections/community/general/lvg_module.html)

Waiting for migration or backporting, as workaround the solution is to add non-destructive pvresize command managed by a var to respect backward compatibility and allowing complete size extension automation (avoiding manual execution on hosts).

**Comment by Roxyrob** [comment]
Fixed in #57
